RobotsAndPencils/Eureka
# Update to Eureka4.1.1 and Add KVO Fixes

This PR updates our master to Eureka's master and applies our changes, which are:

Pat Holmvigs KVO fixes for deleting rows
The same fixes Pat did, but for deleting sections
Note: We should really try to get these changes into the official Eureka repo, so we don't have to keep doing this.

To Test

Just make sure everything compiles and maybe makes sure the examples run.
If they are in tests (are there?), make sure those run.

You can also verify that the Caliber H2O app works with these changes (The latest PR for that will have to be approved first)
